keps/sig-node/NNNN-pass-resource-reqs-limits-to-cri/README.md

```diff
@@ -196,35 +229,25 @@ spec:
 ```
 
 Kubelet constructs `ResourceConfig`, sketched here in `yaml` format:
-```
+```yaml
 PodResourceConfig:
   containers:
     - name: db
```

Comment:
Now there are no more list or `name` fields under the `PodResourceConfig`. Instead container names (I suppose) are keys in the map under the `init_containers` and `containers` fields. So this example needs an update match the new form...

```yaml
PodResourceConfig:
  containers:
    db:
      requests:
        cpu: 1900m
        mem: 10G
      limits:
        cpu: 1900m
        mem: 10G
    db-sync-with-hw-accel:
      requests:
        cpu: 100m
        memory: 100M
        intel.com/qat: 2
      limits:
        cpu: 100m
        memory: 100M
        intel.com/qat: 2        
```

I think this is a nice simplification and improves the original design. Originally I had an idea to leave room for backwards compatible extension of new fields related to resource requests - maybe something like topology wishes. But thinking it now, I don't think it's worth the complexity at this point. And there definitely are ways to add that information than extending originally proposed `ContainerResource` with new fields next to `resource` and `value`.

Reply:
> I think this is a nice simplification and improves the original design. Originally I had an idea to leave room for backwards compatible extension of new fields related to resource requests - maybe something like topology wishes. But thinking it now, I don't think it's worth the complexity at this point. And there definitely are ways to add that information than extending originally proposed `ContainerResource` with new fields next to `resource` and `value`.

Thanks for noting this, I'll update the PR.

The reason I dropped the `ContainerResource` message is that I think it might be better to just let the pod spec resources "fall-through" as is. If extensions are needed in the future maybe they should just be added as new fields to `ContainerResourceConfig`(?)

The reason I changed `init_containers` and `containers` in `PodResourceConfig` from arrays to maps was to drop the `name` field from `ContainerResourceConfig` because that is redundant information in `ContainerConfig` and `UpdateContainerResourcesRequest` messages.

Reply:
> Instead container names (I suppose) are keys in the map under the `init_containers` and `containers` fields

PR updated and this is now fixed